src/signing/adapters/types.ts

```diff
@@ -25,7 +25,7 @@ export interface PRESIGN_RESULT {
  */
 export interface Adapter {
   type: string
-  hash: string
+  HASHING_ALGORITHM: string
```

Comment:
:fire: - I am pretty sure this is a mistake?
Out lack of testing in this module is _dangerous_

```suggestion
  hash: string
```

Reply:
tldr; this is a documentation issue not a code one